src/bin/pg_autoctl/monitor_pg_init.c

```diff
@@ -117,6 +126,28 @@ monitor_pg_init(Monitor *monitor)
 }
 
 
+/*
+ * replace_monitor_pg_hba gets the path of pgData of the monitor, and
+ * overrides the pg_hba rules.
```

Comment:
I think this comment is a good place to explain two things:

  1. why we don't like the default rules in pg_hba.conf and how the different --auth-local and --auth-host options can't help us there,
  2. what we expect to see as our own default rules in pg_hba.conf in the monitor instead